jquery/webpack.config.js

```diff
@@ -30,6 +30,13 @@ Encore
         config.corejs = '3.23';
     })
     .enableSassLoader()
-;
+    .enablePostCssLoader();
 
-module.exports = Encore.getWebpackConfig();
+module.exports = {
+    ...Encore.getWebpackConfig(),
+    watch: true,
+    watchOptions: {
+    aggregateTimeout: 200,
+    poll: 1000
+    }
```

Comment:
```suggestion
    watchOptions: {
      aggregateTimeout: 200,
      poll: 1000
    }
```

Utiliser une variable d'environnement pour `watch` & `poll` si on a pas envie de l'activer (ça utilise beaucoup de cpu).